app/_data/docs_nav_konnect.yml

```diff
@@ -75,6 +75,8 @@
               url: /runtime-manager/configuration/vaults/how-to/
         - text: Manage Runtime Configuration using decK
           url: /runtime-manager/declarative-config/
+    - text: KIC Integration
```

Comment:
❗ Reviewers: I chose to add this here because it seemed to me that these aren't just runtime instances, though maybe putting this under "Runtime Configuration" would also work or be better?